src/plugins/customStyles/index.js

```diff
@@ -0,0 +1,32 @@
+import { CUSTOM_BLOCK_STYLES,CUSTOM_TEXT_STYLES } from './constants';
+import Immutable from 'immutable';
+import styles from './customStyles.css';
+
+const createCustomStylesPlugin = () => {
+
+    const blockRenderMap = Immutable.Map(CUSTOM_BLOCK_STYLES);
+    const customStyleMap = CUSTOM_TEXT_STYLES;
+
+    const blockStyleFn = (contentBlock) => {
+        let style = [];
+
+        const type = contentBlock.getType();
+        for(var key in CUSTOM_BLOCK_STYLES) {
+            if(CUSTOM_BLOCK_STYLES.hasOwnProperty(key)) {
```

Comment:
Inutile, tu parcours déjà les clé de ton object en faisant le for(var key in ...), il contient forcément ta clé